openmicroscopy/openmicroscopy
# Float rendering web

Some small changes to support floating point data ranges in the channel sliders in Preview tab and main image viewer.
NB: this won't work fully until re.getChannelWindowStart/End(i) returns floats without rounding.

To test:
- Open a float pixel-type image in Preview panel. Test various images with very small and large dynamic ranges.
- Start positions of slider handles are Integers (see above) but sliding handles should give a reasonable number of floating point increments (100 currently for full range of slider).
- Save will save floating point start/end to DB (although this can't be verified because of bug above).
- Launch Image Viewer from preview panel should apply the current floating point slider values to the image viewer. 

--rebased-to #3377
